log4j-core/src/test/resources/java-import.ldif

```diff
@@ -0,0 +1,4 @@
+dn: dc=apache,dc=org
```

Comment:
Shall we rename this to `JndiLdapLookupTest.ldif` instead to make the association more clear?

Reply:
Yeah - I will rename it to match the name of the test class.